### DISCUSSION TITLE
Copy links on Android mobile app

### DISCUSSION THREAD

**Discussion by Araxeus** [community/community#134929] [Mobile]
If there is a link in a README, or an issue etc

There should be an option to long press the link to copy it

Right now it seems like the only option is to click the link and open it in the default browser

https://github.com/user-attachments/assets/47c3c069-e831-4003-bd92-813eab961a89

**Comment by stevepopovich** [comment]
Hi @Araxeus are you on Android or iOS?

**Reply by Araxeus** [reply]
Android (and i've added above a short video to show the issue)

**Reply by stevepopovich** [reply]
Ah yeah, fair. Ok, I can file this as a feature request, thanks!

**Comment by PhilC813** [comment]
Definitely a small but noticeable missing capability
